nilearn/plotting/tests/test_surf_plotting.py

```diff
@@ -360,6 +365,9 @@ def test_plot_surf_roi():
     # plot roi list
     plot_surf_roi(mesh, roi_map=[roi1, roi2])
 
+    # plot to axes
+    plot_surf_roi(mesh, roi_map=roi1, ax=plt.gca())
```

Comment:
plt.gca() doesn't has 3d projection which we essentially need for plotting. Isn't it ?